pkg/status/status.go

```diff
@@ -35,3 +35,8 @@ func GenerateBuildMsg(stage Stage, state State, msg string) string {
 	statusMsg := []string{string(stage), string(state), msg}
 	return strings.Join(statusMsg, "|")
 }
+
+func GenerateWorkflowStatusMsg(stepPosition int32, state State, errorCode string, msg string) string {
```

Comment:
i don't think `string(stepPosition)` works as expected, use fmt.Sprintf instead

Reply:
not working either 
strconv.Itoa(int(stepPosition))